tests/e2e/utils/flows.js

```diff
@@ -589,43 +587,6 @@ export const merchantWCP = {
 		} );
 	},
 
-	openDisputeDetails: async ( disputeDetailsLink ) => {
-		await Promise.all( [
-			page.goto( WC_ADMIN_BASE_URL + disputeDetailsLink, {
-				waitUntil: 'networkidle0',
-			} ),
-			uiLoaded(),
-		] );
-		await uiLoaded();
-	},
-
-	openChallengeDispute: async () => {
```

Comment:
I've removed this helper function and moved the logic inline to simplify the code.